tests/slow/src/test/scala/tests/sbt/SbtSlowSuite.scala

```diff
@@ -200,19 +200,15 @@ object SbtSlowSuite extends BaseImportSuite("sbt-import") {
       _ <- server.initialize(
         s"""
            |/project/build.properties
-           |sbt.version=1.2.6
+           |sbt.version=1.3.0
            |/build.sbt
            |scalaVersion := "2.12.10"
-           |lazy val a = project.settings(scalaVersion := "2.12.4")
```

Comment:
removed 2.12.4 since it seemed to get stuck on it - checking what happens if we remove it